cms/templates/widgets/sock.html

```diff
@@ -15,41 +17,11 @@
 
   <div class="wrapper-inner wrapper">
     <section class="sock" id="sock" aria-labelledby="sock-heading">
-        <h2 id="sock-heading" class="title sr-only">${_("{studio_name} Documentation").format(studio_name=settings.STUDIO_NAME)}</h2>
+      <h2 id="sock-heading" class="title sr-only">${_("{studio_name} Documentation").format(studio_name=settings.STUDIO_NAME)}</h2>
+      <%
+        links = get_sock_links()
```

Comment:
this is defined here to separate function call from for loop below for easier debug if ever necessary.